src/store/actions/actions.js

```diff
@@ -0,0 +1,23 @@
+export const DELETE_BOOK = "DELETE_BOOK";
+export const deleteBook = (payload) => {
+  return {
+    type: DELETE_BOOK,
+    payload,
+  };
+};
+
+export const CREATE_BOOK = "CREATE_BOOK";
+export const createBook = (payload) => {
+  return {
+    type: CREATE_BOOK,
+    payload,
+  };
+};
+
+export const UPDATE_BOOK = "UPDATE_BOOK";
+export const updateBook = (payload) => {
+  return {
+    type: UPDATE_BOOK,
+    payload,
+  };
+};
```

Comment:
В целом все норм, но надо избегать большое количество экспортов, это делает код менее читабельным.
Например помести actions в отдельный объект.
```suggestion
export const BOOK_ACTIONS = {
  DELETE_BOOK: "DELETE_BOOK",
  CREATE_BOOK: "CREATE_BOOK",
  UPDATE_BOOK: "UPDATE_BOOK",
};
```

Reply:
Так же переименнуй файл в book.actions.js так как название файла должно говорить о какой то сущности